File: factorial/java/Factorial.java
import java.util.Scanner;

public class Factorial {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter a number : ");
        int input = scanner.nextInt();

        int userInput = input;
        long factorial = 1;

        while (input > 0) {
            factorial = factorial * input;
            input = input - 1;
        }
        
        System.out.println("Factorial of " + userInput + " is " + factorial);

        scanner.close();
    }

}

File: factorial/python/Factorial.py
print('Enter a number: ')
user_input = int(input());

original_input = int(user_input) 
factorial = 1

while (user_input > 1):
    factorial = factorial * user_input
    user_input = user_input - 1

print('Factorial of ' + str(original_input) + ' is ' + str(factorial))